homeworks/src/homework8/FileLogger.java

```diff
@@ -0,0 +1,69 @@
+package homework8;
+
+import java.io.*;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+public class FileLogger {
+    /*
+    Клас здійснюватиме логування (протоколування) інформації
+    в зазначений файл на підставі конфігураційного об'єкта.
+    */
+    private FileLoggerConfiguration file;
+
+    public FileLogger(FileLoggerConfiguration file) {
+        this.file = file;
+    }
+
+    public void info(String message) throws FileMaxSizeReachedException {
+        File fo = new File(file.getFileName());
+        if ((int )fo.length() > file.maxFileSize) {
+            throw new FileMaxSizeReachedException(file.fileName, file.maxFileSize, fo);
+        }
```

Comment:
What will happened, when this check is passed, but oldFile + newLogs > file.maxFileSize?